src/test/scala/fable/ConsumerSpec.scala

```diff
@@ -184,6 +184,28 @@ class ConsumerSpec extends AsyncFunSuite {
     }).unsafeToFuture
   }
 
+  test("listTopics") {
+    val fableTopic = Topic("fable-test")
+    val consumer = Consumer.resource[IO, String, String](config)
+
+    (for {
+      _ <- createTopic(fableTopic.name)
+      topicsAndPartitions <- consumer.use { instance =>
+        for {
+          _ <- instance.subscribe(fableTopic)
+          _ <- instance.poll
```

Comment:
So if we don't poll here for some reason the topic doesn't get returned in from our listTopics method. Not sure why that would be?

Reply:
`KafkaConsumer` is kind of weird in when it fetches data. I had the same issue with partitions, and I resolved it by using `poll`. It would be nice to find another solution, but I haven't learned one yet.